mparticle/models/device_information.py

```diff
@@ -909,6 +927,66 @@ def bluetooth_version(self, bluetooth_version):
 
         self._bluetooth_version = bluetooth_version
 
+    @property
+    def amp_id(self):
+        return self._amp_id
+
+    @amp_id.setter
+    def amp_id(self, amp_id):
+        self._amp_id = amp_id
+
+    @property
+    def roku_advertising_id(self):
+        return self._roku_advertising_id
+
+    @roku_advertising_id.setter
+    def roku_advertising_id(self, roku_advertising_id):
```

Comment:
note that the s2s DTO only marks Roku Ad ID and Fire Ad ID as Guids so those are the only settings that get the validation treatment.